### ISSUE TITLE
SwiftnessX Icon is pixelated on Windows 10.

### ISSUE THREAD

**Issue by Testerpixie** [ehrishirajsharma/SwiftnessX#59]

**Comment by webdevterri** [comment]
Agreed. I am on Win10 as well, and the icon has a very fuzzy look to it. 
![BlurrySwiftnessIcon](https://user-images.githubusercontent.com/32005382/101118950-d2f8c600-35af-11eb-9454-f9e58753abf4.png)
